src/main/java/Parser.java

```diff
@@ -0,0 +1,99 @@
+import java.util.Scanner;
+
+/**
+ * Parses the inputs given by the user.
+ */
+public class Parser {
+    /**
+     * Determines the keyword and processes the input based on the keyword.
+     */
+    static String parse(String inp) {
+        String[] spl = inp.split(" ", 2);
```

Comment:
Could perhaps use a actual word to name the variable spl, as it is being used throughout the parse code

Reply:
also for the checkSplLength below

Reply:
Thank you for your review :)